Makefile

```diff
@@ -19,7 +26,19 @@ docker-push:
 docker-run:
 	docker run -it --rm --net host ${DOCKER_TAG}
 
+
+
+
+
 # Start MongoDB server in container
```

Comment:
Instead of writing instructions just make another target named `db-shell` or `db-cli` which runs the required commands.

```
docker run -it --rm --net host mongo:latest mongo -u ${DB_USER} -p ${DB_PASSWORD}
```